doc/conf.py

```diff
@@ -0,0 +1,172 @@
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+from sys import path as sys_path
+from os.path import abspath
+from pathlib import Path
+from json import loads
+
+
+ROOT = Path(__file__).resolve().parent
+
+sys_path.insert(0, abspath('.'))
+sys_path.insert(0, abspath('..'))
+
+
+# ==============================================================================
+# Project information
+# ==============================================================================
+project =   "pyEDAA.CLITool"
```

Comment:
It's not using `pyTooling.Packaging`.
Need to be changed in `pyEDAA.CLITool` too.

Reply:
Done (for both).